docs/statistics.md

```diff
@@ -314,266 +472,230 @@ either already acknowledged or dropped by TSBPD as too late to be delivered.
 
-## byteRcvLoss
+### byteRcvLoss
 
-Same as `byteRcvLossTotal`, but for a specified interval.
+Same as `byteRcvLossTotal`, but for a specified interval. Available for receiver.
 
-## byteRetrans
+### byteRetrans
 
-Same as `byteRetransTotal`, but for a specified interval.
+Same as `byteRetransTotal`, but for a specified interval. Available for sender.
 
-## byteSndDrop
+### byteSndDrop
 
-Same as `byteSndDropTotal`, but for a specified interval.
+Same as `byteSndDropTotal`, but for a specified interval. Available for sender.
 
-## byteRcvDrop
+### byteRcvDrop
 
-Same as `byteRcvDropTotal`, but for a specified interval.
+Same as `byteRcvDropTotal`, but for a specified interval. Available for receiver.
 
-## byteRcvUndecrypt
+### byteRcvUndecrypt
 
-Same as `byteRcvUndecryptTotal`, but for a specified interval.
+Same as `byteRcvUndecryptTotal`, but for a specified interval. Available for receiver.
 
-# Instant measurements
 
-The measurements effective at the time retrieved.
+## Instantaneous Statistics
 
-## usPktSndPeriod
+### usPktSndPeriod
 
-Current minimum time interval between which consecutive packets are sent, in 
-microseconds. Sender only.
+TODO: 
 
-Note that several sockets sharing one outgoing port use the same sending queue.
-They may have different pacing of the outgoing packets, but all the packets will
-be placed in the same sending queue, which may affect the send timing.
+- How is this calculated? The minimum time during which period?
+- probing packets
+- rephrase - They may have different pacing of the outgoing packets, but all the packets will
+  be placed in the same sending queue, which may affect the send timing. - who may have sockets? which may affect?
 
-`usPktSndPeriod` is the minimum time (sending period) that must be kept
+The current minimum packet inter-sending time, in microseconds. Does not take into account the probing packet pairs. Available for sender.
+
+`usPktSndPeriod` is the minimum time (minimum sending period) that must be kept
 between two packets sent consecutively over the link used by an SRT socket.
 It is not the EXACT time interval between two consecutive packets. In the case where the time spent by an 
 application between sending two consecutive packets exceeds `usPktSndPeriod`, the next 
 packet will be sent faster, or even immediately, to preserve the average sending rate.
 
-**Note**: Does not apply to probing packets.
+Note that several SRT sockets sharing one outgoing port use the same sending queue.
+They may have different pacing of the outgoing packets, but all the packets will
+be placed in the same sending queue, which may affect the send timing.
+
+### pktFlowWindow
 
-## pktFlowWindow
+TODO:
 
-The maximum number of packets that can be "in flight". Sender only.
-See also [pktFlightSize](#pktFlightSize).
+- Rephrase this - The maximum number of packets that can be "in flight" state. - it does not reflect the idea
+- revise the whole paragraph
```

Comment:
"In flight" means the packets that have departed the sender, but did not arrive at the receiver, in a certain point in time. Precise estimation of this value is impossible without having some 3rd independent probe machine that can retrieve data at precisely the same time from both machines simultaneously, but this value can at least exist in theory. The only point in time when this value can be calculated closely to reality is at the moment when ACK is being received by the data sender, and it's the distance between ACK and the top sender sequence - though this still includes the number of packets that have been received during the time when the ACK packet was travelling through the network.

In this value this is the "maximum" allowed flight size - because the sender can only control those packets that it didin't sent yet, and therefore if the measured flight size exceeds the maximum allowed flight size, it must stop sending, even though the receiver buffer free space didn't get to 0.